vaxrank/core_logic.py

```diff
@@ -49,6 +49,16 @@ def vaccine_peptides_for_variant(
         variant=variant,
         protein_sequence=isovar_protein_sequences[0])
 
+    min_expected_length = mhc_predictor.min_peptide_length
+    def_expected_length = mhc_predictor.default_peptide_lengths
+    if def_expected_length is not None and len(def_expected_length) > 0:
```

Comment:
@armish thanks for the deep dive! Seems like we should at least be using the min in a nontrivial (read: not with a value of 1) way inside mhctools, especially since there's all this length checking in the stack. Agreed that this deserves a refactor at some point (I'm happy to do that too) but yeah for now, we should fix in vaxrank. Would you be okay with moving the check to inside the vaxrank epitope prediction code though, so that we explicitly catch these mhctools errors?